### PR TITLE
mrc-1543 dockerise app

### PR DESCRIPTION
**Pull request by hillalex**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,12 +23,28 @@ Requirements:
...
+
+```
+./scripts/build-app.sh
+```
+ 
+This is run for each branch as part of the Travis build and the resulting image pushed to Dockerhub. 
+To run:
 
+```
+docker run -p 8080:8080 mrcide/hint:branch_name
```
Comment (EmmaLRussell) [inline README.md:43]: Replace hint with mint..

File: README.md [hunk 2]
```diff
@@ -14,12 +23,28 @@ Requirements:
...
+```
 
+or
 
+```
+docker run -p 8080:8080 mrcide/hint:git_sha
```
Comment (EmmaLRussell) [inline README.md:49]: ..and here too

**Comment by hillalex** [comment]
> ` docker run -p 8080:8080 mrcide/mint:mrc-1543` should work shouldn't it?
> 
> I get `docker: Error response from daemon: manifest for mrcide/mint:mrc-1543 not found: manifest unknown: manifest unknown.`
> Has travis not built the image?
> 
> Oh, this is in the log: `"docker push" requires exactly 1 argument`
> https://travis-ci.com/github/mrc-ide/mint/jobs/322060230
> Guess it failed silently?

ah I think it's because it's in the after_success step, so it doesn't fail the build. moved to scripts step now (and fixed the script)

**Comment by hillalex** [comment]
Cool looks like it worked now!

**Pull request merged**